spec/ddtrace/contrib/graphql/tracer_spec.rb

```diff
@@ -32,6 +31,8 @@
     end
 
     describe 'query trace' do
+      before { skip('GraphQL does not support ddtrace 1.0. Code must be updated in their package.') }
+
       subject(:result) { schema.execute(query, variables: {}, context: {}, operation_name: nil) }
 
       let(:query) { '{ foo(id: 1) { name } }' }
```

Comment:
I think should we print a warning when customers try to use this integration with 1.0, if we end up releasing without upstream having a fix in place.

Also, this seems like something to be mentioned in https://github.com/DataDog/dd-trace-rb/pull/1832 .

Reply:
Yeah, will add it to the guide. Should probably add a compatibility check to the integration itself; only issue is there's no compatible version for now, so might not be able to do this until they've updated.

Reply:
Since we already know what the API will look like, should we also start engaging with them on this right now? Graphql seems to be common enough amongst our customers, and it seems a bit weird that the 1.0.0 stable release is the one that breaks it :)

Reply:
Sure, you could start a PR, but it won't make any sense to ask it to be merged/reviewed if it doesn't actually work against a non-released version. I expect the turnaround on this will be pretty quick, even if there might be a little time before they release their own package.

Reply:
We'll work closely with the upstream GraphQL gem with the appropriate changes as soon as 1.0 is released.

Reply:
I think we should do it before we release 1.0. We know what the API will look like, right? 

Aren't we just going to tell them to probe the them version and either use `Datadog.tracing.trace` or `Datadog::Tracing.trace` depending on < 1 or >= 1?

🤔  I'm probably missing something here.

Reply:
Makes sense, I captured that in our internal tracker.